BeVolt/Periphs/src/I2C.c

```diff
@@ -48,10 +48,16 @@ uint8_t I2C3_Read(uint8_t deviceAddr, uint16_t regAddr){
 	return data;
 }
 
-void I2C3_ReadMultiple(uint8_t deviceAddr, uint16_t startAddr, uint8_t *rxData, uint32_t rxSize){
+ErrorStatus I2C3_ReadMultiple(uint8_t deviceAddr, uint16_t startAddr, uint8_t *rxData, uint32_t rxSize){
+	int timeout_threshold = 240000000;
```

Comment:
This should probably be either `#defined` or declared `const`, especially since all the functions use the same. Some information about how long the delay (in the form of a comment, etc.) would also be useful.